src/player.cpp

```diff
@@ -10102,6 +10102,20 @@ int player::get_wind_resistance(body_part bp) const
     return totalCoverage;
 }
 
+int player::get_weight() const
+{
+    int ret = 0;
+    int wornWeight = 0;
+    for (auto itm : worn) {
```

Comment:
This line makes a full copy of an `item`. Pretty expensive. I'd also suggest `std::accumulate` instead of a for loop.